### PR TITLE
Update CONTRIBUTING, add SAVED_REPLIES

### PR DESCRIPTION
**Pull request by filipesilva**
Based on the ones present in angular/angular:
- https://github.com/angular/angular/blob/master/CONTRIBUTING.md
- https://github.com/angular/angular/blob/master/docs/SAVED_REPLIES.md

We should all use these saved replies while triaging to make the process easier, more consistent and more informative for users.

/cc @IgorMinar

### CODE REVIEW

File: .github/SAVED_REPLIES.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Saved Responses for Angular CLI's Issue Tracker
+
+The following are canned responses that the Angular CLI team should use to close issues on our issue tracker that fall into the listed resolution categories.
+
+Since GitHub currently doesn't allow us to have a repository-wide or organization-wide list of [saved replies](https://help.github.com/articles/working-with-saved-replies/), these replies need to be maintained by individual team members. Since the responses can be modified in the future, all responses are versioned to simplify the process of keeping the responses up to date.
+
+
+## Angular CLI: Already Fixed (v1)
+```
+Thanks for reporting this issue. Luckily it has already been fixed in one of the recent releases. Please update to the most recent version to resolve the problem.
```
Comment (delasteve) [inline .github/SAVED_REPLIES.md]:
```
Luckily, it has already been fixed in one of the recent releases.
```

File: .github/SAVED_REPLIES.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+If after upgrade the problem still exists in your application please open a new issue and provide a simple repository reproducing the problem and describing the difference between the expected and current behavior. You can use `ng new repro-app` to create a new project where you reproduce the problem.
```
Comment (delasteve) [inline .github/SAVED_REPLIES.md]:
```
If the problem persists in your application after upgrading, please open a new issue, provide a simple repository reproducing the problem, and describe the difference between the expected and current behavior.
```

File: .github/SAVED_REPLIES.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+```
+
+## Angular CLI: Don't Understand (v1)
+```
+I'm sorry but we don't understand the problem you are reporting.
```
Comment (delasteve) [inline .github/SAVED_REPLIES.md]:
```
I'm sorry, but we don't understand the problem you are reporting.
```

File: .github/SAVED_REPLIES.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+If the problem still exists please open a new issue and provide a simple repository reproducing the problem and describing the difference between the expected and current behavior. You can use `ng new repro-app` to create a new project where you reproduce the problem.
```
Comment (delasteve) [inline .github/SAVED_REPLIES.md]:
```
If the problem persists, please open a new issue, provide a simple repository reproducing the problem, and describe the difference between the expected and current behavior.
```

File: .github/SAVED_REPLIES.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+```
+
+
+## Angular CLI: Duplicate (v1)
+```
+Thanks for reporting this issue. However this issue is a duplicate of an existing issue #<ISSUE_NUMBER>. Please subscribe to that issue for future updates.
```
Comment (delasteve) [inline .github/SAVED_REPLIES.md]:
```
However, this issue is a duplicate of an existing issue #<ISSUE_NUMBER>. Please subscribe to that issue for future updates.
```

File: .github/SAVED_REPLIES.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+## Angular CLI: Insufficient Information Provided (v1)
+```
+Thanks for reporting this issue. However, you didn't provide sufficient information for us to understand and reproduce the problem. Please check out [our submission guidelines](https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-submitting-an-issue) to understand why we can't act on issues that are lacking important information.
+
+If the problem still persists, please file a new issue and ensure you provide all of the required information when filling out the issue template.
+```
+
+## Angular CLI: Issue Outside of Angular CLI (v1)
+```
+I'm sorry but this issue is not caused by Angular CLI. Please contact the author(s) of project <PROJECT NAME> or file issue on their issue tracker.
```
Comment (delasteve) [inline .github/SAVED_REPLIES.md]:
```
I'm sorry, but this issue is not caused by Angular CLI.
```

File: .github/SAVED_REPLIES.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+```
+
+
+## Angular CLI: Non-reproducible (v1)
+```
+I'm sorry but we can't reproduce the problem following the instructions you provided.
```
Comment (delasteve) [inline .github/SAVED_REPLIES.md]:
```
I'm sorry, but we can't reproduce the problem following the instructions you provided.
```

File: .github/SAVED_REPLIES.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+If the problem still exists please open a new issue following [our submission guidelines](https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-submitting-an-issue).
```
Comment (delasteve) [inline .github/SAVED_REPLIES.md]:
```
If the problem persists, please open a new issue following [our submission guidelines](https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-submitting-an-issue).
```

File: .github/SAVED_REPLIES.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+```
+Thanks for reporting this issue. However this issue is a duplicate of an existing issue #<ISSUE_NUMBER>. Please subscribe to that issue for future updates.
+```
+
+
+## Angular CLI: Insufficient Information Provided (v1)
+```
+Thanks for reporting this issue. However, you didn't provide sufficient information for us to understand and reproduce the problem. Please check out [our submission guidelines](https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-submitting-an-issue) to understand why we can't act on issues that are lacking important information.
+
+If the problem still persists, please file a new issue and ensure you provide all of the required information when filling out the issue template.
```
Comment (delasteve) [inline .github/SAVED_REPLIES.md]:
```
If the problem persists, please file a new issue and ensure you provide all of the required information when filling out the issue template.
```

File: .github/SAVED_REPLIES.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+I'm sorry but we can't reproduce the problem following the instructions you provided.
+
+If the problem still exists please open a new issue following [our submission guidelines](https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-submitting-an-issue).
+```
+
+## Angular CLI: Obsolete (v1)
+```
+Thanks for reporting this issue. This issue is now obsolete due to changes in the recent releases. Please update to the most recent Angular CLI version.
+
+If the problem still persists, please file a new issue and ensure you provide the version of Angular CLI affected and include the steps to reproduce the problem when filling out the issue template.
```
Comment (delasteve) [inline .github/SAVED_REPLIES.md]:
```
If the problem persists after upgrading, please open a new issue, provide a simple repository reproducing the problem, and describe the difference between the expected and current behavior.
```

**Comment by filipesilva** [comment]
Fixed all grammar improvements highlighted in @delasteve's review.

**Pull request merged**